features/schema.py

```diff
@@ -68,6 +68,15 @@ class FeatureSource(ObjectType):
     )
 
 
+class PriceTag(DjangoObjectType):
+    class Meta:
+        model = models.PriceTag
+
+    item = graphene.String()
+    price = graphene.Decimal()
```

Comment:
if these two cannot be `null`, you could set the scalars to `...(required=True)` to show that in the schema.

Reply:
fixed